src/html.rs

```diff
@@ -357,7 +354,7 @@ pub fn print_dom(handle: &Handle) {
 }
 
 fn is_icon(attr_value: &str) -> bool {
-    ICON_VALUES.is_match(&attr_value.to_lowercase())
+    ICON_VALUES.contains(&attr_value)
```

Comment:
If there is no to_lowercase, then rel="Shortcut Icon" won't match, it has to be case-insensitive here. I've seen it having capital letters on at least one rather popular web resource. I should've made a test for that when I found the issue.